.travis.yml

```diff
@@ -23,6 +23,7 @@ cache:
     - /tmp/depcache
 
 before_install:
+  - nvm install 10.15.1
```

Comment:
why do we need to install nvm?

Reply:
its needed for latest smart contracts

Reply:
Im not installing NVM btw, its already there on travis. Im just installing the latest version of nodejs.